backend/tracim_backend/lib/utils/utils.py

```diff
@@ -345,5 +347,21 @@ def is_dir_readable(path: str) -> bool:
     if not os.access(
         path=path, mode=os.R_OK | os.X_OK, dir_fd=None, effective_ids=os.supports_effective_ids
     ):
-        raise NotReadableDirectory("{} is not a writable directory".format(path))
+        raise NotReadableDirectory("{} is not a readable directory".format(path))
+    return True
+
+
+def is_file_exist(path: str) -> bool:
```

Comment:
why do you re-implement `os.path.isfile()` ?

Reply:
by the way, i suggest to rename this function to `file_exists()`

Reply:
reason of this code is : 
- to return a specific exception
- to have similar methods as other checking methods (is_dir_exist, is_dir_writable, is _dir_readable, etc...)

I'm not sure about renaming, current name is explicit about returning boolean if ok, and other methods have similar naming. Should we change naming of all theses utils methods ?